docs/project/contribution_tools.md

```diff
@@ -130,8 +130,9 @@ These tools are essential for work on Carbon.
             outdated, and not be upgraded.
 -   Main tools
     -   [Bazel](https://www.bazel.build/)
+        -   At least bazel version 6.0.0.
     -   [Bazelisk](https://docs.bazel.build/versions/master/install-bazelisk.html)
-        (for macOS): Handles Bazel versions.
+        (for macOS, Linux, Windows): Handles Bazel versions.
```

Comment:
```suggestion
        (for macOS): Handles Bazel versions.
```

This documentation is explaining which tools are used according to the [install instructions above](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/contribution_tools.md#debian-or-ubuntu). While Bazelisk works on multiple platforms, and could be used instead of the instructions on this page, this list of tools is expressly what is "essential for work on Carbon", and Bazelisk is only essential on macOS.